### PR TITLE
Portfolio: setup and mobile first 

### PR DESCRIPTION
**Pull request by CobbyElsonfx**
# Added the following changes 
1. Changed  the "h2" tag  to  "a"  tag for the logo section
2. Used and "h1" tag for the introductory section "I am a Cobby and ...."
3. Made use ul li for the  social media icons list
4. Removed the "header" serving as a wrapper for the whole  section and wrapped it only around the Navbar
5. Added a section tag to the headline section
6. Made sure the social media icons were aligned well
7. Made use of the right font-weight for each sections 
8. Added some css to the background image of the headline to make it look like the figma style

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                            <img src="./assets/git.png" alt="git">
+                            <img src="./assets/twohandicon.png" alt="my pub">
+                        </div> 
+
+                    </div>
+                                          
+                </div>
+            </div>
+        </div>
+    </header>
```
Comment (usorfaitheloho) [inline index.html]: You did well with the toolbar section, however, this `header` section should end on line 24, please the close `header` should be on line 24 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    <header>
+        <div class="hero">
+            <nav class="navbar">
+                <h2 class="logo">Cobby Elson</h2>
+                <button type="button" class="hamburger-btn">
+                    <span class="toggler-icon top-bar"></span>
+                    <span class="toggler-icon middle-bar"></span>
+                    <span class="toggler-icon bottom-bar"></span>
+                </button>
+            </nav>
```
Comment (usorfaitheloho) [inline index.html]: Kindly implement these changes for the toolbar section to have a better UI and a better user experience :

-  Your logo on line 17 should be in an `<a href =""></a>` tag and not an `h2` tag for good semantics as it is shown in your code 😄 
-  The hamburger button should be in a `nav` tag for semantics reasons, and the right position for convention and user experience for mobile devices

File: index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                            <img src="./assets//Linkedin icon.png" alt="linkedin">
+                            <img src="./assets/socialIcon.png" alt="social">
+                            <img src="./assets/git.png" alt="git">
+                            <img src="./assets/twohandicon.png" alt="my pub">
+                        </div> 
+
+                    </div>
+                                          
+                </div>
+            </div>
```
Comment (usorfaitheloho) [inline index.html]: Kindly implement these changes for your `headline` section 👍 

1. The container of this section should be in a `section` tag as it represents a generic standalone section of the document, you can check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more details
2. You did well adding some background images, kindly include all background images as your design lacks some of them 😄 
3. The heading on line 27 should be in a `h1` tag for search engine optimization about a specific topic. you can check this [link](https://www.reliablesoft.net/h1-tag/) for more details
4. The list of icons should be in  a `ul li` tag like this
   > 
         <ul>
                   <li> twitter</li>
                             .
                             .
                             .
           </ul>

5. Kindly use the right font-color for the paragraph `i'm a software developer which is color `#344563`, 
6. For all your texts in this section, please use the right font properties as given in the figma design so as to have a better user design and experience 😄 
7. Your social media icon doesn't seem to be properly aligned in a straight line as some has a higher width than others, could you please correct that 😃 

Figma design :
image

Design given :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @CobbyElsonfx  :thumbsup:  ,

This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are still some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linters are passing ✔️ 
-  Well-structured files ✔️
<br/>
<br/>
<br/>

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (CobbyElsonfx) [reply index.html]: @usorfaitheloho  Thanks  please,  but with the second comment thus 

- The hamburger button should be in a nav tag for semantics reasons, and the right position for convention and user experience for mobile devices
- The logo and button are already in a nav tag, so should i strip that one off and use a div  in place of that whiles I use a nav tag only for the button?

File: index.html (hunk 1 above)
Comment (CobbyElsonfx) [reply index.html]: @usorfaitheloho Alright I understand ,so after putting the header element on line 24. 
- Should i also put the left over code into a section tag ?
- Must there be a wrapper for the whole  header and the next section (like wrapping both components with a div)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+.about-section {
+  padding-top: 20%;
+  width: 97%;
+  height: 100%;
+  background-image: url("./assets/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
```
Comment (aamirkhan2478) [inline style.css]: - [x] As mentioned by the previous reviewer :point_up_2: please stick to the Figma design but still your design is not same as Figma design if you check your design the background image is not matched with Figma design so kindly fix it :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face: 

**Your Design**
Screenshot from 2023-06-01 23-39-39

**Figma design**
Screenshot from 2023-06-01 23-39-15

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @CobbyElsonfx , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                        </div>
+                        <div class="about-section-item links">
+                            <a href="http://" class="lets-connect">LET’S CONNECT</a>
+                            <ul class="social-media-icons">
+                                <li><img class="twitter" src="./assets//twitter.png" alt="twitter"></li>
+                                <li><img class="link" src="./assets//Linkedin icon.png" alt="linkedin"></li>
+                                <li><img class="socialIcon" src="./assets/socialIcon.png" alt="social"></li>
+                                <li><img class="git" src="./assets/git.png" alt="git"></li>
+                                <li><img class="twohandicon" src="./assets/twohandicon.png" alt="my pub"></li>
+                            </ul>
```
Comment (ademibowale) [inline index.html]: - [ ] [optional] i will mark this as optional suggestion as your previouse reviewer didnt state this foryou, please i would suggest you to use `a` tag in all your socail icon  ,according to the project requirement The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) for proper semantic reason.

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href please fix this .

Screenshot 2023-06-02 014043

**Review by ademibowale — APPROVED** [review]
Hi @CobbyElsonfx  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**